conf/application.conf

```diff
@@ -197,4 +197,397 @@ versioning {
+                "AK",
+                "AL",
+                "AM",
+                "AN",
+                "AP",
+                "AQ",
+                "AR",
+                "AS",
+                "AT",
+                "AU",
+                "AV",
+                "AW",
+                "AX",
+                "AY",
+                "AZ",
+                "BA",
+                "BB",
+                "BC",
+                "BD",
+                "BE",
+                "BF",
+                "BG",
+                "BH",
+                "BI",
+                "BJ",
+                "BP",
+                "BQ",
+                "BR",
+                "BS",
+                "BT",
+                "BU",
+                "BV",
+                "BW",
+                "BX"
+            ]
+        }
+        "read:individuals-income-laa-c4" {
+            fields = [
+                "A",
+                "B",
+                "C",
+                "D",
+                "E",
+                "F",
+                "G",
+                "J",
+                "AJ",
+                "AP",
+                "BC",
+                "BD",
+                "BE",
+                "BF"]
+        }
+
+        "read:individuals-income-hmcts-c2" {
+            fields = ["H", "J", "K", "M", "N", "O", "R", "T", "U", "AI", "AP", "BC", "BD", "BE", "BF", "BG"]
+        }
+
+        "read:individuals-income-hmcts-c3" {
+            fields = ["H", "J", "K", "M", "N", "O", "R", "T", "U", "AI", "AP", "BC", "BD", "BE", "BF", "BG"]
+        }
+
+        "read:individuals-income-hmcts-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "J", "AJ", "AO", "AP", "BC", "BD", "BE", "BF"]
+        }
+
+        "read:individuals-income-lsani-c1" {
+            fields = ["I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "AA", "AB", "AF", "AH", "AI", "AL", "AM", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "AZ", "BA", "BB", "BC", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BK", "BL", "BM", "BN", "BO", "BP"
+        }
+
+        "read:individuals-income-lsani-c3" {
+            fields = ["I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "AA", "AB", "AF", "AH", "AI", "AL", "AM", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "AZ", "BA", "BB", "BC", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BK", "BL", "BM", "BN", "BO", "BP"
+        }
+
+        "read:individuals-employments-nictsejo-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "N", "O", "P", "Q", "R", "S", "T", "V", "W", "X", "AJ", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "BC", "BD", "BE", "BF", "BH", "BI", "BJ", "BK", "BL", "BM", "BN"]
+        }
+    }
+
+    endpoints {
```

Comment:
Also this endpoint config looks different to that in the test config used bu the code for testing the handler pattern. 

```
    endpoints = List(
      EndpointConfig(
        name = mockEndpoint1,
        link = "/a/b/c?matchId=<matchId>{&fromDate,toDate}",
        fields = Map(
          "A" -> "payments",
          "B" -> "employer/employerName=\"foo bar\"",
          "C" -> "employer/employerAddress/line1",
          "D" -> "employer/employerAddress/line2",
          "E" -> "employer/employerAddress/line3",
          "F" -> "employer/employerDistrictNumber",
          "G" -> "employer/employerSchemeReference",
          "H" -> "employmentStartDate",
          "I" -> "employmentLeavingDate",
          "J" -> "employmentPayFrequency",
          "K" -> "employeeAddress",
          "L" -> "payrollId"
        )
      ),
```

I think we will need to align to this name, link, fields 

Name is the string name of the endpoint used by the endpoint to fetch the scopes. 

Link is the full url for HAL 

Fields are the fields. 

@owen-opencast  can help confirm this?